src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.Emitter.cs

```diff
@@ -88,6 +91,26 @@ public InterceptsLocationAttribute(string filePath, int line, int column)
                         }
                     }
                     """);
+                }
+                else
+                {
+                    _writer.WriteLine($$"""
+                    namespace System.Runtime.CompilerServices
+                    {
+                        using System;
+                        using System.CodeDom.Compiler;
+
+                        {{Expression.GeneratedCodeAnnotation}}
+                        [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+                        file sealed class InterceptsLocationAttribute : Attribute
+                        {
+                            public InterceptsLocationAttribute(int version, string data)
```

Comment:
>(int version, string data [](http://example.com/codeflow?start=62&length=25)

nit: looks this is the only different part when version is 1. would it better to store it in a string template with {} and pass the filling value according to InterceptorVersion?

Reply:
Sure thanks